### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by machacekondra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,150 +1,102 @@
...
+[cloud.vmware_ops.provision_vcenter](https://github.com/redhat-cop/cloud.vmware_ops/blob/main/playbooks/provision_vcenter/provision_vcsa_on_esxi.yml)|A playbook to provison Vcenter on ESXi host.
+[cloud.vmware_ops.provision_vm](https://github.com/redhat-cop/cloud.vmware_ops/blob/main/playbooks/provision_vm/manage_vm.yml)|A playbook to create/manage virtual machines on VMware.
+[cloud.vmware_ops.system_settings](https://github.com/redhat-cop/cloud.vmware_ops/blob/main/playbooks/system_settings.yml)|A playbook to manage system settings on VMware.
+<!--end collection content-->
 
-## External requirements
+## Installation and Usage
 
-<!-- List any external resources the collection depends on, for example minimum versions of an OS, libraries, or utilities. Do not list other Ansible collections here. -->
+### Requirements
```
Comment (bardielle) [inline README.md:38]: I think it's worth to ada: 
vmware.vmware_rest

python packages:
pyvmomi

and maybe assign a section on EE, what do you think?

File: README.md [hunk 2]
```diff
@@ -1,150 +1,102 @@
...
 
-See the [changelog](https://github.com/ansible-collections/REPONAMEHERE/tree/main/CHANGELOG.rst).
 
-## Roadmap
+## Contributing to this collection
+
+We welcome community contributions to this collection. If you find problems, please open an issue or create a PR against this collection repository.
 
-<!-- Optional. Include the roadmap for this collection, and the proposed release/versioning strategy so users can anticipate the upgrade/update cycle. -->
+### Testing and Development
```
Comment (bardielle) [inline README.md:82]: Please add the integration tests as well

File: README.md (hunk 1 above)
Comment (machacekondra) [reply README.md:38]: `vmware.vmware_rest` is mentioned. Not sure if we should specify dependencies of dependency, in that case we would need to specify also `http-asyncio` etc.

> and maybe assign a section on EE, what do you think?

You mean to document EE? We can add it, but it's not needed as part of validation process. So let's add it later.

Comment (machacekondra) [reply README.md:38]: Also I see other collections don't metion the dependency collection dependencies as well:
https://github.com/redhat-cop/cloud.gcp_ops?tab=readme-ov-file#requirements

**Review by bardielle — APPROVED** [review]

**Pull request merged**